### ISSUE TITLE
USB Keyboard not working?

### ISSUE THREAD

**Issue by andreacampanella** [opencardev/crankshaft#167]
trying the latest NG with a one plus 3 on a raspberry pi 2+

If I connect the keyboard , it works in the configuration screen but no into the the OpenAuto screen.
[As from documentation ](https://github.com/opencardev/openauto/blob/4fa68d1d7c977cccc2c4586b39f893cf9994113a/src/autoapp/UI/SettingsWindow.cpp#L237) some keys should work but apparenly are not ?

**Comment by smartroad** [comment]
Possibly a stupid suggestion, but have you enabled the keyboard in the settings?

**Comment by andreacampanella** [comment]
Possibly a stupid answer: which settings ? :P

**Comment by smartroad** [comment]
There is a tab in the settings which (if memory serves) there is a check box in the top left to enable global keyboard commands and then you need to check which ones you want to have enabled individually.

**Comment by andreacampanella** [comment]
Not in the NG , anyway , the keyboard works in Crankshaft but not in AA

**Comment by hawkeyexp** [comment]
I checked it - usb keyboard is working - no config needed - for example play pause.
If something won't work with usb keyboard it must be adressed on openauto by f1x.
CS has no changes in this code part.

https://github.com/f1xpl/openauto/wiki/Keyboard-button-bindings

**Comment by Bkmz2017** [comment]
Help! I have same problem, keyboard doesnt work in OpenAuto screen, but when I reboot it, I see in console my presing. And keyboard work in configuration screen and in console.
Actually, it worked when I just installed Crancshaft and worked perfectly, then something happened and it doesnt work. I have changed raspberry, keyboard, sd-card and update Crancshaft image. It still no working. I have no idea why. 
And of course I have enabled the keyboard in the settings.
